smarts/core/vehicle_index.py

```diff
@@ -71,6 +75,18 @@ def social_vehicle_ids(self):
         ]["vehicle_id"]
         return set(vehicle_ids)
 
+    def actor_is_boid(self, actor_id):
```

Comment:
I would suggest that it should be the role of the agent manager to keep track of if the actor is a boid. I can see why this would be useful though.